api/buildings/migrations/0001_initial.py

```diff
@@ -7,43 +7,87 @@ class Migration(migrations.Migration):
 
```

Comment:
Migrations are auto-generated code and should be whitelisted in `black`. We don't want it messing around with these.

Reply:
```
black --line-length 80 --target-version py37 --exclude migrations ./api
```